.github/workflows/lib-checks.yml

```diff
@@ -115,44 +92,4 @@ jobs:
         id: cache-dir-mypy
         run: mkdir -p -v .mypy_cache
       - name: "Run mypy"
-        run: mypy --non-interactive discord/
-  pytest:
```

Comment:
Note: despite being called pytest this didn't run any tests but rather just ran the linting checks which is now done by ruff